GLE_continuation_v0.py

```
@@ -0,0 +1,857 @@
+  # Run continuation
+  print(f"\nStarting pseudo-arclength continuation:")
+  print(f"  mu_r = {args.mu_r}")
+  print(f"  lambda_slip = {args.lambda_slip}")
+  print(f"  theta0 = {args.theta0}°")
+  print(f"  Target Ca = {args.Ca_target}")
+  print()
+  
+  start_time = time.time()
+  
+  try:
+    branch = solver.pseudo_arclength_continuation(Ca_target=args.Ca_target)
```

Ruff (0.11.9)

Local variable `branch` is assigned to but never used

Remove assignment to unused variable `branch`

(F841)